associations/views/page.py

```diff
@@ -9,11 +9,10 @@ class PageViewSet(AssociationNestedViewSet):
     serializer_class = PageSerializer
     permission_classes = (PagePermission,)
 
-    def get_queryset(self):
-        return Page.objects.filter(association=self.kwargs['association_pk'])
+    def perform_create(self, serializer):
+        # Send the current user to the serializer so they can be added to the authors.
+        serializer.save(author=self.request.user)
 
     def perform_update(self, serializer):
-        serializer.save(association=Association.objects.get(pk=self.kwargs['association_pk']))
-
-    def perform_create(self, serializer):
-        serializer.save(association=Association.objects.get(pk=self.kwargs['association_pk']))
+        # Send the current user to the serializer so they can be added to the authors.
+        serializer.save(author=self.request.user)
```

Comment:
Does it work to says that `author` is a specific user since it's a many to many relationship ? Should be make something like `serializer.instance.authors.add(self.request.user)` ?

Reply:
The `serializer.save` method was overrided:
```python
    def save(self, author, **kwargs):
        """Override the save method to add `author` to the authors field."""
        instance = super(PageSerializer, self).save(**kwargs)
        instance.authors.add(author)
        return instance
```